### ISSUE TITLE
RangeError: Index out of range while scanning mp3 files with version 2.0.0

### ISSUE THREAD

**Issue by quantumkv** [Borewit/music-metadata#100]
**Bug description**
While trying to scan a library of mp3 files, the following error occurs:
```
Uncaught (in promise) RangeError: Index out of range
    at checkOffset (buffer.js:977)
    at Buffer.readUInt32BE (buffer.js:1051)
    at Function.FrameParser.readData (FrameParser.js:131)
    at Function.ID3v2Parser.readFrameData (ID3v2Parser.js:97)
    at ID3v2Parser.parseMetadata (ID3v2Parser.js:200)
    at eval (ID3v2Parser.js:154)
    at <anonymous>
```

This error occurs on version 2.0.0. Scanning the same library with version 1.1.0 works without any issue.

**Expected behavior**
The Files are scanned correctly without throwing any errors.

**Comment by Borewit** [comment]
Blocks: webtorrent/webtorrent-desktop#1418

**Comment by Borewit** [comment]
A familiar face. Thanks for letting me know Tarak.
Can you please provide me a sample?

**Comment by quantumkv** [comment]
I've attached 2 out of some 20 files triggering the error.

[Test.zip](https://github.com/Borewit/music-metadata/files/2079285/Test.zip)

**Comment by Borewit** [comment]
Fixed in [v2.0.1](https://github.com/Borewit/music-metadata/releases/tag/v2.0.1).

Thanks for reporting, I consider the issue as fixed, confirmation would be great.

**Comment by quantumkv** [comment]
Works perfect with v2.0.1
